dspot/src/main/java/eu/stamp_project/dspot/selector/PitMutantScoreSelector.java

```diff
@@ -36,19 +35,22 @@
 
     private int numberOfMutant;
 
-    private List<PitResult> originalKilledMutants;
+    private List<AbstractPitResult> originalKilledMutants;
 
-    private Map<CtMethod, Set<PitResult>> testThatKilledMutants;
+    private Map<CtMethod, Set<AbstractPitResult>> testThatKilledMutants;
 
-    private List<PitResult> mutantNotTestedByOriginal;
+    private List<AbstractPitResult> mutantNotTestedByOriginal;
+
+    private AbstractParser parser;
 
     public PitMutantScoreSelector() {
         this.testThatKilledMutants = new HashMap<>();
+        parser = new PitXMLResultParser();
```

Comment:
Here, the parser is hardcoded. Could we use a command line option or a property to allow the user to use the format he wants, XML or CSV?